src/models/Activity.js

```diff
@@ -22,16 +23,22 @@ const activitySchema = new mongoose.Schema(
           "holiday",
           "cultural",
         ],
-        message: "Invalid category type!",
+        message:
+          "Invalid category type! Choose from ['relaxing', 'creative', 'social', 'educational', 'holiday', 'cultural']",
       },
       required: [true, "please provide a valid category!"],
     },
-    votes: {
-      type: Number,
-      default: 0,
-    },
+    votes: { type: mongoose.Schema.Types.ObjectId, ref: "Vote" },
```

Comment:
See the comment below about `voteSchema`. If we make that change, this would change to be a calculated field.

Reply:
This is interesting, because there should only be the `votes: { type: mongoose.Schema.Types.ObjectId, ref: "Vote" },` portion and not anything else that you are pointing in here in the Activity data model file. And I am not entirely sure what you mean by a calculated field.

Reply:
Ah, I was thinking of this as a vote count, but yeah, I see that it's just a reference to the vote object. 👍